tests/js/click/click-basic-tests.js

```diff
@@ -113,22 +90,18 @@ https://github.com/fluid-lab/gamepad-navigator/blob/master/LICENSE
 
         jqUnit.test("Checkboxes can be selected by click.", function () {
             gamepad.tests.windowObject.navigator.getGamepads = function () {
-                return gamepad.tests.utils.element.click(0);
+                return gamepad.tests.utils.element.click(0, gamepad.tests.navigator);
             };
 
             // Initialize the webpage, i.e., focus on the checkbox element.
             $("#checkbox-one").focus();
 
             // Confirm that the instance of the gamepad navigator is created.
             gamepad.tests.navigator = gamepad.inputMapper({
-                windowObject: gamepad.tests.windowObject
+                windowObject: gamepad.tests.windowObject,
+                members: { count: 2 }
```

Comment:
Rather than pass this as option, I'd suggest defining a gamepad.tests.inputMapper grade that has the members and windowObject setting (if possible).